crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -8,35 +8,27 @@ use crate::syntax::{
     attribute::{attribute_find_path, attribute_tokens_to_ident},
     path::path_to_single_ident,
 };
-use std::collections::HashMap;
+use std::collections::BTreeMap;
 use syn::{spanned::Spanned, Error, Ident, Item, ItemEnum, ItemImpl, Result, Type, TypePath};
 
 #[derive(Default)]
 pub struct ParsedCxxQtData {
     /// Map of the QObjects defined in the module that will be used for code generation
-    pub qobjects: HashMap<Ident, ParsedQObject>,
+    pub qobjects: BTreeMap<Ident, ParsedQObject>,
     /// The namespace of the CXX-Qt module
     pub namespace: String,
     /// Any `use` statements end up in the CXX-Qt generated module
     pub uses: Vec<Item>,
 }
 
 impl ParsedCxxQtData {
-    /// Find the QObjects within the module and add into the qobjects HashMap
+    /// Find the QObjects within the module and add into the qobjects BTreeMap
     pub fn find_qobject_structs(&mut self, items: &[Item]) -> Result<()> {
         for item in items {
             if let Item::Struct(qobject_struct) = item {
                 if let Some(index) =
                     attribute_find_path(&qobject_struct.attrs, &["cxx_qt", "qobject"])
                 {
-                    // TODO: for now we only support one qobject per block
```

Comment:
Nice to see all it took was to remove a few lines :D

Reply:
Right, most of the other stages were tested to be working, we just didn't know if there were any other missing parts :-)